src/cheri_insts.sail

```diff
@@ -974,54 +933,26 @@ union clause ast = CUnseal : (regidx, regidx, regidx)
  * Capability register *cd* is replaced with capability register *cs1* and is
  * unsealed, using capability register *cs2* as the authority for the unsealing
  * operation. If *cs2*.**perms** does not grant **Global** then *cd*.**perms**
- * is stripped of **Global**.
- *
- * ## Exceptions
- *
- * An exception is raised if:
- *   - *cs2*.**tag** is not set.
- *   - *cs1* is not sealed.
- *   - *cs2* is sealed.
- *   - *cs1*.**otype** is reserved.
- *   - *cs2*.**address** $\ne$ *cs1*.**otype**.
- *   - *cs2*.**perms** does not grant **Permit_Unseal**.
- *   - *cs2*.**address** $\lt$ *cs2*.**base**.
- *   - *cs2*.**address** $\ge$ *cs2*.**top**.
+ * is stripped of **Global**. If *cs2* is unable to authorize the unsealing,
+ * the **tag** field of *cd* is cleared.
```

Comment:
Almost surely worth a comment giving justification?  Something like... "We believe it to be simpler and more flexible for compartmentalized software to check the tag of the result of unsealing an input capability rather than immediately trapping if the otype is out of range.  If software omits such a check and directly uses the result of unsealing for memory dereference, a malformed input will cause a tag-clear trap on the dereference instead."  Probably needs some workshopping.

Reply:
Do you mean inline in the comment and latex, or in the rationalisation section? I imagine the ISA doc will grow a detailed section on tag-clearing vs trapping decisions (initial draft lives [here](https://github.com/CTSRD-CHERI/cheri-specification/pull/1) that needs expansion, especially to talk about sealing). Comments in the sail are otherwise very rationale-light, and I guess we'd need this comment on Seal, Unseal, and all variant friends?

Reply:
Yeah, on second thought, you're right; the rationale belongs elsewhere in the ISA doc rather than in the more mechanically-minded sail comments.